near-sdk-macros/src/core_impl/abi/abi_embed.rs

```diff
@@ -1,8 +1,17 @@
-use proc_macro2::TokenStream as TokenStream2;
+use proc_macro2::{Span, TokenStream as TokenStream2};
 use quote::quote;
 
 pub fn embed() -> TokenStream2 {
-    let abi_path = env!("CARGO_NEAR_ABI_PATH");
+    let abi_path = match option_env!("CARGO_NEAR_ABI_PATH") {
+        Some(path) => path,
+        None => {
+            return syn::Error::new(
+                Span::call_site(),
+                "CARGO_NEAR_ABI_PATH environment variable is not set",
```

Comment:
Can you improve this error so it indicates that the `__abi-embed` feature should not be set, and is only used through cargo-near? This error isn't any more descriptive than the previous one

Reply:
Was just about asking for error message suggestions. What do you think?

- <code>\`__abi-embed\` feature flag should not be set manually, use \`cargo near\`</code>
- <code>invalid use of private feature flag \`__abi-embed\`, please remove this feature</code>

And if we're mentioning the use of `cargo near`, should we add a link to it?

Reply:
up to you on that, either sounds great! And yeah, I think a link to it would be useful

Reply:
Went through a couple of error message iterations and settled on this:

<img width="614" alt="CleanShot 2022-08-30 at 20 59 56@2x" src="https://user-images.githubusercontent.com/16881812/187496348-4965255e-39dd-427f-af63-cce614bae3b2.png">

Would've loved to add the hint into the notes to more seamlessly blend in with cargo / rustc style error reports but the interface for that is yet to be stabilized. https://github.com/rust-lang/rust/issues/54140